functions/openhabian.bash

```diff
@@ -24,6 +24,7 @@ get_git_revision() {
 ##
 install_cleanup() {
   echo -n "$(timestamp) [openHABian] Cleaning up... "
+  clean_config_userpw
```

Comment:
That should be run directly from `openhabian-setup.sh`, right?

Reply:
Right, removed it here. It's already called from openhabian-setup.sh.
Frankly I do not recall my original intention for moving it.